firmware/Core/Src/telecommands/telecommand_parser.c

```diff
@@ -279,7 +279,38 @@ uint8_t TCMD_parse_full_telecommand(const char tcmd_str[], TCMD_TelecommandChann
         return 90;
     }
 
-    // Reached the end of the telecommand parsing. Thus, success. Fill the output struct.
+    // ensure the corrent number of args are provided
+    int32_t num_args_expected = TCMD_telecommand_definitions[tcmd_idx].number_of_args;
+
+    char error_message[80];
+    snprintf(error_message, 80, "Error: TCMD_parse_full_telecommand: this telecommand acepts %d arguments.\n", num_args_expected);
```

Comment:
2 things:

1. I think you should be able to say the actual name of the telecommand here.
2. Spelling mistake for accept.

Reply:
3. Use `sizeof(error_message)` as the second arg to `error_message` (best practice to avoid repeating magic numbers).